### PR TITLE
Save failed screen

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/save_failed_screen.c [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    return TRUE;
+}
+
+static bool16 VerifySectorWipe(u32 sector)
+{
+    u16 sector0 = sector;
+    u16 i;
+    u32 *saveDataBuffer = (void *)&gSaveDataBuffer;
+    ReadFlash(sector0, 0, saveDataBuffer, 0x1000);
+    for (i = 0; i < 0x1000 / 4; i++, saveDataBuffer++)
```
Comment (jiangzhengwenjz) [inline src/save_failed_screen.c]: `sizeof(u32)`

File: src/save_failed_screen.c [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    {
+        if (*saveDataBuffer != 0)
```
Comment (jiangzhengwenjz) [inline src/save_failed_screen.c]: `NULL`

Comment (PikalaxALT) [reply src/save_failed_screen.c]: Read it again.

Comment (jiangzhengwenjz) [reply src/save_failed_screen.c]: fuck me

**Pull request merged**